tests/hare/config.yaml

```diff
@@ -4,28 +4,28 @@ config_map_name: conf
 
 script_on_exit: './k8s/log-client-pods.sh'
 
-genesis_delta: 30
-deployment_ready_time_out: 90
+genesis_delta: 90
+deployment_ready_time_out: 120
 
 bootstrap:
-  image: 'spacemeshos/go-spacemesh-dev:hare19'
+  image: 'spacemeshos/go-spacemesh-dev:hare31'
   command: '/bin/go-hare'
   replicas: 1
   args:
-    hare-committee-size: '6'
-    hare-max-adversaries: '3'
-    hare-round-duration-sec: '10'
-    layer-duration-sec: '120'
+    hare-committee-size: '50'
+    hare-max-adversaries: '25'
+    hare-round-duration-sec: '60'
+    layer-duration-sec: '360'
 
 client:
-  image: 'spacemeshos/go-spacemesh-dev:hare19'
+  image: 'spacemeshos/go-spacemesh-dev:hare31'
```

Comment:
You should have the final image in spacemeshos/go-spacemesh:hare

Reply:
Fixed